_posts/2020-05-29-fault-tolerance-4-2-and-4-3.adoc

```diff
@@ -0,0 +1,72 @@
+:page-layout: post
+:page-title: Fault Tolerance 4.2 & 4.3
+:page-synopsis: SmallRye Fault Tolerance 4.2 and 4.3 released!
+:page-tags: [announcement, microprofile]
+:page-date: 2020-05-29 12:00:00.000 +0100
+:page-author: lthon
+
+= Fault Tolerance 4.2 & 4.3
+
+Today, we announce two new releases of SmallRye Fault Tolerance: 4.2.1 and 4.3.0.
+
+== 4.2.1
+
+This release includes bugfixes and component updates and is safe to use for everyone who uses Fault Tolerance 4.1.0, 4.1.1 and 4.2.0.
+The 4.2.0 release happened a few weeks ago and didn't contain anything but bugfixes either.
+With 4.2.1, we now use MicroProfile Fault Tolerance 2.1.1, which includes some TCK fixes.
+
+There is one small feature in the 4.2.1 release, which is interesting for integrators.
+If you integrate SmallRye Fault Tolerance and find you need to provide your own `ExecutorFactory` because you need SmallRye Fault Tolerance to use different thread pools, you can now inherit from `DefaultExecutorFactory` and override the `threadFactory` method.
```

Comment:
`If you integrate SmallRye Fault Tolerance and find you need` replace by `If you integrate SmallRye Fault Tolerance and you need`?

`because your need` replace by `because`? (`you need` repeated in the same sentence)

Reply:
Did a more significant rewording.